index.html

```diff
@@ -51,16 +51,19 @@
       <h2>Introduction</h2>
 
 <p>W3C is a <strong>growing and global community</strong> where participants
-choose to work together, and in that process experience differences in
-language, location, nationality, and experience. In such a diverse environment,
+choose to work together, and it is committed to maintaining a positive working environment, where each participant feels appreciated and respected and where everyone adheres to the same high level of standards of personal behavior. In that process we experience differences in language, location, nationality, and experience. In such a diverse environment, 
 misunderstandings and disagreements happen, which in most cases can be resolved
-informally. In rare cases, however, behavior can intimidate, harass, or
-otherwise disrupt one or more people in the community, which W3C will not
```

Comment:
I think we want to keep the sense here that W3C will act "formally" in cases where problems are not resolved. I don't think "will not tolerate" is the right wording, though I am not sure what is.

Reply:
4/4/19 PWECG agreed to this change